instruqt/delivering-as-an-appliance/05-downloading-and-installing/assignment.md

```diff
@@ -0,0 +1,143 @@
+4. **Airgap Download Enabled** Allows the customer to download the air-gapped
+   version of the Embedded Cluster installer.
+2. **Support Bundle Upload Enabled** Configures their license to allow
+   uploading support bundles directly from the Admin Console for an online
+   installation.
+2. **Allow Snapshot** Includes components for application-level [backup and
+   restore](https://docs.replicated.com/vendor/snapshots-overview).
+2. **Embeded Cluster Enabled** Allows the customer to download the Embedded
+   Cluster installer and install an Embedded Cluster instance.
+4. **Allow Disaster Recovery** Provides disaster recovery functionality for
+   the Embedded Cluster.
+
+We don't need all of these options for this lab. We'll just enable the two
+that we require: "KOTS Install Enabled" and "Embedded Cluster Enabled". Scroll
+down to "License options" make sure both are active. You'll need to enable
+KOTS first.
+
+![Setting license options for the Embedded Cluster](../assets/embedded-cluster-license-options.png)
+
+Once you've enabled the options, click "Save changes" and scroll back to the
+top of the page. You may have noticed a button that said "Helm Install
+Instructions" ealier. It's now change to a menu for "Install instructions".
+Select "Embedded Cluster" to see the instructions to download and install.
+
+![Embedded Cluster Install Instructions](../assets/embedded-cluster-install-instructions.png)
+
+Running the Installer
+=====================
+
+You can use the instructions from the Vendor Portal to download and install
+SlackerNews using the Replicated Embedded Cluster. Select the "Shell" tab and
+enter the download command from the vendor portal.
+
+```bash
+curl -f https://replicated.app/embedded/[[ Instruqt-Var key="REPLICATED_APP" hostname="node" ]]/stable \
+    -H "Authorization: [[ Instruqt-Var key="LICENSE_ID" hostname="node" ]]" \
+    -o [[ Instruqt-Var key="REPLICATED_APP" hostname="node" ]]-stable.tgz
+```
+
+This will download a tarball including the installer binary and the license
+file for Nitflex. Extract the tarball to access the installer.
+
+```bash
+tar -xzvf [[ Instruqt-Var key="REPLICATED_APP" hostname="node" ]]-stable.tgz
+```
+
+Then run the installer:
+
+```bash
+sudo ./[[ Instruqt-Var key="REPLICATED_APP" hostname="node" ]] install --license license.yaml
+```
+
+The installer will prompt you for a password for the Admin Console. You can
+enter any password you'd like, but you'll need to remember it when you log in
+later. You can copy/paste the password below if you aren't feeling inspired.
+
+```
+[[ Instruqt-Var key="ADMIN_CONSOLE_PASSWORD" hostname="node" ]]
+```
+
+The installer starts by converting the machine you're running on into a single
+node Kubernetes cluster. It will then install several components that support
+Embedded Cluster operation, including the Admin Console that you'll use for
+to complete the installation. You'll see a message and a spinner as each step
```

Comment:
```suggestion
to configure and install the application. You'll see a message and a spinner as each step
```
^ uses the same "configure and install" language that is used in the output of the EC install command